app/jobs/publish_feed_job.rb

```diff
@@ -15,7 +15,8 @@ def perform(podcast, pub_item)
     Rails.logger.info("Starting publishing pipeline via PublishFeedJob", {podcast_id: podcast.id, publishing_queue_item_id: pub_item.id})
 
     PublishingPipelineState.start!(podcast)
-    podcast.feeds.each { |feed| publish_feed(podcast, feed) }
+    podcast.feeds.each { |feed| publish_apple(podcast, feed) }
+    podcast.feeds.each { |feed| publish_rss(podcast, feed) }
```

Comment:
Okay, so here is kind of the crux of it.

It used to be the _public/default feed_ that had a `f.apple_config`.  But now it's tied to the `private_feed.apple_config`.

So - we can't rely on the `podcast.feeds[0]` (the default_feed) processing first, which blocks RSS until we publish to apple.  Instead, we do _all_ the apple publishing first, before moving onto RSS.